packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -297,6 +297,16 @@ class DateTimePicker extends React.Component {
       const previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : DateTimeUtils.createSafeDate(formattedDate, this.initialTimeZone);
       updatedDateTime = DateTimeUtils.syncDateTime(previousDateTime, date, this.timeValue, this.props.showSeconds);
 
+      if (previousDateTime.isDST() && previousDateTime.hours() === 3) {
+        if (!updatedDateTime.isDST() && (event.key === '-' || event.key === '_')) {
```

Comment:
1. Do we need to also handle using the UP and DOWN arrow keys to increment and decrement the day?
2. How about the workflow where the user manually changes the date from 03/08/2020 to 03/09/20200? Should the hour also change back from `03` back to `02`

For example:
Step 1: date/time is `03/09/2020 02:30`
Step 2: Manually change only the day from `03/09/2020` to `03/08/2020`. The hour is auto updated from `02` to `03`.
Step 3: Manually change only the day back to `03/09/2020`. Shouldn't the hour also get reverted back to `02`?

Reply:
> How about the workflow where the user manually changes the date from 03/08/2020 to 03/09/20200? Should the hour also change back from 03 back to 02

Since it works ( resets time to default value 02:30 on changing date using shortcut keys ) when provided the initial date that as missing hour for example : `const initialValue = moment().tz('America/Chicago').format('2020-03-08T02:30:00');` I think it should behave same when user manually enters a date that as missing hour.